src/ImageRunModal.jsx

```diff
@@ -152,6 +253,9 @@ export class ImageRunModal extends React.Component {
                            checked={this.state.hasTTY}
                            onChange={e => this.onValueChanged('hasTTY', e.target.checked)} />
                 </label>
+
+                <label className='control-label'> {_("Ports")} </label>
```

Comment:
you 're missing htmlFor. It's a must for screen readers.

Reply:
Can I just let this point to the `PublishPortsForm` ? Or does this need to point to actual `input` elements?

Reply:
It need to really point to the element you want to select/focus on when clicking on the label. So I think you want input and not the wrapper. But try it, maybe I am wrong.